src/js/info.js

```diff
@@ -0,0 +1,86 @@
+function formatBrand (tags) {
+	const regex = /brand:/i;
+	const tag = tags && tags.find(tag => regex.test(tag));
```

Comment:
`Array.prototype.find` is a new method and should be added the the `browserFeatures.required` array in `origami.json`.